packages/web/src/pages/sign-on/components/desktop/MobileContentContainer.tsx

```diff
@@ -0,0 +1,28 @@
+import { HTMLAttributes, PropsWithChildren } from 'react'
+
+import { Flex, FlexProps } from '@audius/harmony'
+import cn from 'classnames'
+
+import styles from './MobileContentContainer.module.css'
+
+export type MobileContentContainerProps = PropsWithChildren<FlexProps> &
```

Comment:
seems we really only use children + className, can prob just say "FlexProps" ?

Reply:
Yeah I tried just FlexProps but it didnt bring in children for some reason 🤷

Reply:
Oh I see why. It's because the `FlexProps` type doesn't include the React/html props that Emotion adds for us. So `FlexProps` is basically just our custom types

Reply:
yeah i think we should update flexprops type to include